projects/ui-components/lf-metadata/lf-field-template-container/lf-field-template-container.component.html

```diff
@@ -1,15 +1,46 @@
 <div [hidden]="hideTemplate">
   <mat-form-field floatLabel="never" class="lf-field lf-template-picker lf-dropdown">
-    <mat-label class="lf-dropdown-label">{{emptyTemplateName | async}}</mat-label>
-    <mat-select name="template-dropdown" id="template-dropdown" class="lf-dropdown"
-        (selectionChange)="onTemplateChangedAsync($event)"
-        (openedChange)="onToggleDropdownAsync($event)"
-        [(value)]="templateSelected" [compareWith]="compareTemplateInfoFunc">
-        <mat-option selected>{{emptyTemplateName | async}}</mat-option>
-        <mat-option *ngFor="let template of availableTemplates" [value]="template">{{template.name}}
-        </mat-option>
+    <mat-label *ngIf="isDropdownLoading; else checkDropdownError" class="lf-dropdown-label">
+      <lf-loader-component></lf-loader-component>
+    </mat-label>
+    <ng-template #checkDropdownError>
+      <mat-label  *ngIf="isDropdownError; else showDefaultEmptyTemplate" class="lf-dropdown-label">
+        {{ emptyTemplateName | async }}
```

Comment:
what should the mat-label be when getAvaliableTemplatesAsync fails? I was thinking effectively it is still "no template is selected", so showing that instead of saying in the mat-label "an error has occurred" (since this will be shown as a mat-option). which do you think is better?

option 1: label is "no template selected", option is "an error has occurred"

![image](https://user-images.githubusercontent.com/99910777/199844475-8edbd988-4e70-453e-84e6-c2d63933dfc3.png)
![image](https://user-images.githubusercontent.com/99910777/199844529-2a89d1c2-6a4d-4597-8a70-0f38e9fde018.png)
![image](https://user-images.githubusercontent.com/99910777/199844564-6ec1df52-1a10-46c1-913f-8b50be8842e0.png)

option 2: both label and option says "an error has occurred"

![image](https://user-images.githubusercontent.com/99910777/199845147-ae617293-ed07-419b-b02a-26afea64b108.png)
![image](https://user-images.githubusercontent.com/99910777/199845212-6a487ce8-5cda-4222-a0c6-242b7ca21d8d.png)
![image](https://user-images.githubusercontent.com/99910777/199845178-879061a0-9d5f-4a2c-89cb-4078d9ad1310.png)

Reply:
I think it should be No template assigned. For some reason the first time, I thought it had two of them

Reply:
I think option 1 is what we want